### PR TITLE
Add query function calls from secondary buffers.

### PR DESCRIPTION
**Pull request by lentinem**
Issue #722

### CODE REVIEW

**Comment by tobine** [comment]
LGTM :+1:
Verified passing on farm

File: tests/layer_validation_tests.cpp [hunk 1]
```diff
@@ -4731,6 +4731,137 @@ TEST_F(VkLayerTest, WaitEventThenSet) {
     m_errorMonitor->VerifyNotFound();
 }
 // This is a positive test.  No errors should be generated.
+TEST_F(VkLayerTest, QueryAndCopySecondaryCommandBuffers) {
+    TEST_DESCRIPTION(
+        "Issue a query on a secondary commad buffery and copy it on a primary.");
```
Comment (chrisforbes) [inline tests/layer_validation_tests.cpp]: secondary command buffer ?

File: tests/layer_validation_tests.cpp [hunk 2]
```diff
@@ -4731,6 +4731,137 @@ TEST_F(VkLayerTest, WaitEventThenSet) {
...
+    vkAllocateCommandBuffers(m_device->device(), &command_buffer_allocate_info,
+                             &command_buffer);
+
+    VkCommandBuffer secondary_command_buffer;
+    command_buffer_allocate_info.level = VK_COMMAND_BUFFER_LEVEL_SECONDARY;
+    vkAllocateCommandBuffers(m_device->device(), &command_buffer_allocate_info,
+                             &secondary_command_buffer);
+
+    VkQueue queue = VK_NULL_HANDLE;
+    vkGetDeviceQueue(m_device->device(), m_device->graphics_queue_node_index_,
```
Comment (chrisforbes) [inline tests/layer_validation_tests.cpp]: Does this actually need to use a different queue?

File: tests/layer_validation_tests.cpp (hunk 1 above)
Comment (lentinem) [inline tests/layer_validation_tests.cpp]: Done.

File: tests/layer_validation_tests.cpp (hunk 2 above)
Comment (lentinem) [inline tests/layer_validation_tests.cpp]: No. This just submits a single command buffer onto the queue.

**Comment by tobine** [comment]
cherry-picked and pushed to master

**Pull request closed without merging**